src/app/pages/shell/shell.component.ts

```diff
@@ -97,6 +85,20 @@ export class ShellComponent implements OnInit, OnChanges, OnDestroy {
     }
   }
 
+  onRightClick(): false {
+    this._snackbar.open(
+      T(
```

Comment:
I like how this works but I think we may want to get docs opinion on the message, also if we cannot display OS specific cut and paste (since it is different for Mac OS vs Windows and Unix) we should maybe tell the user something like "If you are on Mac OS use Control C and Control V, otherwise use Ctrl Insert and Shift Insert"

Reply:
Checkout #2434